node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -560,9 +561,10 @@ func (o GetDmsPathHelperGeneric) WaitForDmToExist(volumeUuid string, volumeNguid
 	formatTemplate := strings.Join([]string{"%d", "%w"}, mpathdSeparator)
 	args := []string{"show", "maps", "raw", "format", "\"", formatTemplate, "\""}
 	var err error
+	logger.Debugf("Waiting for dm to exist, executing command : {%v} with args : {%v}.", multipathdCmd, args)
 	for i := 0; i < maxRetries; i++ {
 		err = nil
-		out, err := o.executer.ExecuteWithTimeout(TimeOutMultipathdCmd, multipathdCmd, args)
+		out, err := o.executer.ExecuteWithTimeout(TimeOutMultipathdCmd, multipathdCmd, args, false)
```

Comment:
from past experience in debugging customer logs, the output of this command is important to understand which DMs exist at each time

Reply:
done